core/src/translate/mod.rs

```diff
@@ -146,6 +146,25 @@ pub fn translate(sql_statement: &SqlStatement) -> Result<Statement> {
                 "VERSION" => Ok(Statement::ShowVariable(Variable::Version)),
                 v => Err(TranslateError::UnsupportedShowVariableKeyword(v.to_string()).into()),
             },
+            (3, Some(keyword)) => match keyword.value.to_uppercase().as_str() {
+                //   AST: [ShowVariable { variable: [Ident { value: "indexes", quote_style: None },
+                //     Ident { value: "from", quote_style: None }, Ident { value: "mytable", quote_style: None }] }]
```

Comment:
For the PRs which are ready for review, it is recommended to have less comments (except for doc comment).
This looks better not to have these.

Reply:
ok.. done!